apps/web/pages/booking/[uid]/join/[hash].tsx

```diff
@@ -0,0 +1,88 @@
+import type { GetServerSideProps } from "next";
```

Comment:
This one again is a core page and it's importing bigblue directly. I still need to testing how BBB works but we need an abstraction here.

A similar requirement can come from other apps as well.